client/src/components/dataList/SpeciesInfo.js

```diff
@@ -0,0 +1,293 @@
+import { useEffect, useState } from "react";
+import React from "react";
+import { useReducer } from "react";
+// import DeleteSpecies from "./DeleteSpecies";
+import deleteIcon from "./deleteIcon.png"
+// create useReducer for changing values
+import EditSpecies from "./edit/EditSpecies";
+
+const reducer = (state, action) => {
+    console.log(action, 'this is the action');
+    switch (action.type) {
+        case 'editID':
```

Comment:
to be fair, id is usually immutable and you *would not* want to let anyone edit the id.